CONTRIBUTING.md

```diff
@@ -161,13 +161,11 @@ following events:
 
 ### On pushes to `main` branch
 
-_(Will be replaced to "on pushes to `main` branch" after v3 is released.)_
-
 **Workflow:** [`push.yml`](https://github.com/apollographql/apollo-android/blob/main/.github/workflows/push.yml)
 
 **Job:**
 
-- `push`
+- `publish`
```

Comment:
I've just renamed this job to `deploy`, should we keep `deploy`?

Reply:
Oh haha 😅. Well it doesn't matter too much, as long as the name is the same here.